### ISSUE TITLE
naming of "special files"?

### ISSUE THREAD

**Issue by kmindi** [kmindi/special-files-in-repository-root#1]
Is there a more precise term than special files for these special files?

The original name was "Readme-files", but I think this is insufficient to finding them, and gives currently only results for README and Changelog.

**Comment by ghost** [comment]
Agreed, this was the first repository that came up on my google search results.

**Comment by apolopena** [comment]
Six years later and there still is no term ¯\\_(ツ)_/¯
If we invent a term that makes more sense, it could be adopted or maybe just not that many people care about this.
The best I could come up with is **_project meta files_**.
I had to write a command line tool that took an option to move these files to another location so I ended up going with:
`--move-meta-files`
